INSTALLATION.md

```diff
@@ -16,7 +16,7 @@ ext install "Slice"
 To build and package the extension, you must have [`vsce`](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#vsce) installed:
```

Comment:
Why don't we just make `@vscode/vsce` a dev dependency? Do we need it for the build or just for the packaging?

Reply:
It's purely for packaging; after everything has been built, it just zips the files together into a `.vsix` package. That's all.

----

The language server is built to be general purpose.
So, in the future we might support other editors, which would have their own packaging tools.

Reply:
In that case it doesn't need this install step. You can just run with `npx`

Reply:
Oh! I didn't know that cool.
Fixed.

Reply:
You're going to remove it?

Reply:
Whoops, I forgot to push my changes ._.